README.md

```diff
@@ -1,7 +1,22 @@
 # **Observability Remote Write Proxy**
 
-### A proxy to authenticate remote write requests before they are forwarded to Observatorium. 
+### A proxy to accept remote write requests and forward them to Observatorium. Before forwarding the request, the proxy authenticates with the provided oidc credentials and adds the obtained token to the request. The proxy is intended to be used by customers who want to send data to Observatorium but don't want to store Observatorium credentials on their clusters.
+
+## **How to set up the Proxy locally**
+
+#### Flags that are needed to proxy to another prometheus instance:
+--proxy.forwardUrl=<Prometheus remote write URL>
+
+#### Flags that are needed for full functionality:
+--proxy.forwardUrl=<Prometheus remote write URL> --oidc.enabled --proxy.forwardUrl=<Observatorium Forward URL> --oidc.issuerUrl=<OIDC Issuer URL> --oidc.clientId=<OIDC Client ID> --oidc.clientSecret=<OIDC Client Secret> --oidc.audience=<OIDC Audience> --token.verification.enabled --token.verification.url=<Token Verification URL>
```

Comment:
it would be good to describe what exactly those flags do (e.g. what is token verification for and what is oidc for)